website/src/pages/index.tsx

```diff
@@ -32,10 +32,10 @@ function HeroBanner() {
     <div className={styles.hero} data-theme="dark">
       <div className={styles.heroInner}>
         <Heading as="h1" className={styles.heroProjectTagline}>
-          <img
+          <Image
             alt={translate({message: 'Docusaurus with Keytar'})}
             className={styles.heroLogo}
-            src={useBaseUrl('/img/docusaurus_keytar.svg')}
+            img={useBaseUrl('/img/docusaurus_keytar.svg')}
```

Comment:
the ideal image plugin is not designed to work with SVGs

Eventually we could import the svgs and use them as components instead of using `<img src="xyz.svg"/>` like we do atm